modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -104,6 +106,8 @@
 
 			&-text {
 				width: 100%;
+				display: flex;
```

Comment:
If we add the vertical alignment to the `.e-n-tab-text` selector, do we still need it for the `.e-n-tab-icon` as well?

Reply:
Good comment, I the flex from the icon `.e-n-tab-icon`.
Don't see changes.